src/v2/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsTwoFactor/TwoFactorAuthentication/Components/AppSecondFactor/index.tsx

```diff
@@ -41,6 +41,7 @@ export const AppSecondFactor: React.FC<AppSecondFactorProps> = ({
     false
   )
   const [stagedSecondFactor, setStagedSecondFactor] = useState(null)
+  const [confirmedPassword, setConfirmedPassword] = useState("")
```

Comment:
Nit, non-blocking. Do we want to change this a bit since technically this is a password _to be_ confirmed? So maybe something like `passwordToBeConfirmed`, or `passwordConfirmation` to more accurately describe that state of the string?

Reply:
Renamed in [f4f9062](https://github.com/artsy/force/pull/10513/commits/f4f90621b865c57cdf1ab376d6f9a7462e5d1db6)